src/Configuration.php

```diff
@@ -86,4 +115,58 @@ public function getDockerTag(): string
     {
         return $this->dockerTag;
     }
+
+    public function getAppDirectory(): Directory
+    {
+        return $this->appDirectory;
+    }
+
+    public function getProjectDirectory(): Directory
+    {
+        return $this->projectDirectory;
+    }
+
+    public function getVerbose(): bool
+    {
+        return $this->verbose;
+    }
+
+    public function getInspectionProfile(): File
+    {
+        return $this->inspectionProfile;
+    }
+
+    public function setDockerRepository(string $dockerRepository): void
+    {
+        $this->dockerRepository = $dockerRepository;
+    }
+
+    public function setDockerTag(string $dockerTag): void
+    {
+        $this->dockerTag = $dockerTag;
+    }
+
+    /**
+     * @throws ConfigurationException
+     */
+    public function setInspectionProfile(string $inspectionProfile): void
+    {
+        try {
+            $this->inspectionProfile = new File($this->projectDirectory->getPath() . '/' . $inspectionProfile);
```

Comment:
App can know about the inspection profile, just make the createable file object here